### PR TITLE
Fixed port_alias config processing

### PR DESCRIPTION
**Pull request by nazariig**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [] Bug fix
- [] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

### CODE REVIEW

File: ansible/library/port_alias.py [hunk 1]
```diff
@@ -42,30 +42,47 @@
...
+        with open(MACHINE_CONF) as machine_conf:
+            for line in machine_conf:
+                tokens = line.split('=')
+                key = tokens[0].strip()
+                value = tokens[1].strip()
+                if key == ONIE_PLATFORM or key == ABOOT_PLATFORM:
+                    return value
+        return None
+
+    def get_portconfig(self):
```
Comment (mykolaf) [inline ansible/library/port_alias.py]: This function returns portconfig file path, rename it to get_portconfig_path? Current name suggests it parses it imo

File: ansible/library/port_alias.py [hunk 2]
```diff
@@ -42,30 +42,47 @@
...
+        platform = self.get_platform()
+        if platform is None:
+            return None
+        portconfig = FILE_PATH + '/' + platform + '/' + self.hwsku + '/' + PORTMAP_FILE
```
Comment (mykolaf) [inline ansible/library/port_alias.py]: os.path.join?

**Review by mykolaf — CHANGES_REQUESTED** [review]

File: ansible/library/port_alias.py (hunk 1 above)
Comment (nazariig) [reply ansible/library/port_alias.py]: @mykolaf done.

File: ansible/library/port_alias.py (hunk 2 above)
Comment (nazariig) [reply ansible/library/port_alias.py]: @mykolaf done.

**Review by mykolaf — APPROVED** [review]

**Comment by nazariig** [comment]
@stepanblyschak / @andriymoroz-mlnx / @volodymyrsamotiy please review

**Review by stepanblyschak — APPROVED** [review]

**Pull request closed without merging**